cli/request.go

```diff
@@ -43,5 +45,9 @@ func extendRequestWithContent(fs filesystemspec.FileSystem, req controller.Reque
 		req.Units = append(req.Units, controller.Unit{Name: name, Content: content})
 	}
 
+	if len(req.Units) == 0 {
+		return controller.Request{}, errgo.Newf("No unit files found for group '%s'", req.Group)
+	}
+
```

Comment:
We already have request validation: https://github.com/giantswarm/inago/blob/master/controller/validator.go#L87 - does it make sense to use that here?